CleanWeatherTests/Scenes/CityHistorical/FakeCityHistoricalRouter.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  FakeCityHistoricalRouter.swift
+//  CleanWeatherTests
+//
+//  Created by Marek Skrzelowski on 02/12/2019.
+//  Copyright © 2019 vandermesis. All rights reserved.
+//
+
+@testable import CleanWeather
+
+final class FakeCityHistoricalRouter: CityHistoricalRouter {}
```

Comment:
what do you think:

If we have something that is empty, like this router, should we add it to scenes? Empty routers, empty workers etc?
or we should remove it?

I am not sure which one is better
- having empty classes seems to be bad
- but in future it's easier to add features/updates etc

Reply:
I think empty classes should be removed. I left that empty router only because I don't know which direction we're going and if that will be needed for some functionality. When our app will be almost finished I will remove those unnecessary files. Maybe TODO is solution here?